client/client.go

```diff
@@ -274,25 +275,25 @@ func (r *NotaryRepository) initializeRoles(rootKeys []data.PublicKey, localRoles
 			)
 		}
 	}
-	for _, role := range remoteRoles {
+	for _, remoteRole := range remoteRoles {
```

Comment:
Same comment I made above earlier. Changing variable names creates noise in the PR and should really only be done if it makes a significant difference to the readability of the code.